.github/workflows/docker-publish-action.yml

```diff
@@ -1,19 +1,22 @@
-name: Publish API/UI Docker Image
-on: workflow_dispatch
+on:
+  workflow_call:
+    inputs:
+      IMAGE_NAME:
+        required: true
+        type: string
+      DOCKERFILE:
+        required: false
```

Comment:
naïve question: why is `DOCKERFILE` not required?

Reply:
Good catch! It should be required